.gitignore

```diff
@@ -17,3 +17,5 @@ __pycache__/
 **/*.d
 **/*.o
 **/*.su
+
+CMakeLists.txt.user
```

Comment:
Is this needed still? Or was this something you had added for local testing?

Reply:
Not really needed, just useful when you're running tests. I can remove it.

Reply:
Can we remove this if not needed?